packages/button/src/Button.story.tsx

```diff
@@ -34,6 +35,11 @@ storiesOf('Buttons', module)
           null,
         ) as any
       }
+      glyph={
+        <Icon
+          glyph={select('Glyph', Object.keys(glyphs) as Array<any>, 'Edit')}
```

Comment:
We probably should handle the glyph knob the same way in both stories (e.g. as a variable or inline).

I also _think_ that TypeScript should actually infer the type for `Object.keys(glyphs)` as being something like `Array<keyof typeof glyphs>`, so we might not need to typecast it.

Reply:
You are correct, it does not need to be typecast. The glyph knob no longer appears in both stories- Brooke said to just have it in the default component. I wrote it inline because that's how the rest of the knobs were written for the default component, but I can definitely change that if you would prefer to see it in a variable.